components/style/properties/longhand/box.mako.rs

```diff
@@ -275,44 +290,49 @@ ${helpers.single_keyword("overflow-x", "visible hidden scroll auto",
                          gecko_constant_prefix="NS_STYLE_OVERFLOW")}
 
 // FIXME(pcwalton, #2742): Implement scrolling for `scroll` and `auto`.
-<%helpers:longhand name="overflow-y"
-                   need_clone="True"
-                   animatable="False">
-  use super::overflow_x;
+<%helpers:longhand name="overflow-y" need_clone="True" animatable="False">
+    use super::overflow_x;
 
-  use std::fmt;
-  use style_traits::ToCss;
-  use values::computed::ComputedValueAsSpecified;
-  use values::NoViewportPercentage;
+    use std::fmt;
+    use style_traits::ToCss;
+    use values::computed::ComputedValueAsSpecified;
+    use values::NoViewportPercentage;
 
-  pub use self::computed_value::T as SpecifiedValue;
+    pub use self::computed_value::T as SpecifiedValue;
 
-  impl NoViewportPercentage for SpecifiedValue {}
+    impl NoViewportPercentage for SpecifiedValue {}
 
-  impl ToCss for SpecifiedValue {
-      fn to_css<W>(&self, dest: &mut W) -> fmt::Result where W: fmt::Write {
-          self.0.to_css(dest)
-      }
-  }
+    impl ToCss for SpecifiedValue {
+        fn to_css<W>(&self, dest: &mut W) -> fmt::Result where W: fmt::Write {
+            self.0.to_css(dest)
+        }
+    }
 
-  pub mod computed_value {
-      #[derive(Debug, Clone, Copy, PartialEq)]
-      #[cfg_attr(feature = "servo", derive(HeapSizeOf))]
-      pub struct T(pub super::super::overflow_x::computed_value::T);
-  }
 
-  impl ComputedValueAsSpecified for SpecifiedValue {}
+    /// The specified and computed value for overflow-y is a wrapper on top of
+    /// `overflow-x`, so we re-use the logic, but prevent errors from mistakenly
+    /// assign one to other.
```

Comment:
IMO we should actually be using the same value here. There's precedent for having typedefs.